packages/hoppscotch-sh-admin/src/pages/users/index.vue

```diff
@@ -31,16 +31,139 @@
 
         <div v-else-if="error">{{ t('users.load_list_error') }}</div>
 
-        <UsersTable
-          v-else-if="usersList.length >= 1"
-          :usersList="usersList"
-          :fetching="fetching"
-          :error="error"
-          @goToUserDetails="goToUserDetails"
-          @makeUserAdmin="makeUserAdmin"
-          @makeAdminToUser="makeAdminToUser"
-          @deleteUser="deleteUser"
-        />
+        <div v-else-if="usersList.length > 0">
```

Comment:
```suggestion
        <div v-else-if="usersList.length">
```

Won't a truthy check suffice?